corehq/apps/app_manager/models.py

```diff
@@ -3244,6 +3244,36 @@ def get_odk_qr_code(self, with_media=False):
                                          content_type="image/png")
             return png_data
 
+    def generate_shortened_url(self, url_type):
+        try:
+            if settings.BITLY_LOGIN:
+                long_url = get_url_base() + reverse('corehq.apps.app_manager.views.%s' % url_type, args=[self.domain, self._id])
```

Comment:
nit: we're generally trying to use `.format` for string formatting in python. also trying to not to use `+` for concating strings